mutt_commands.c

```diff
@@ -147,18 +129,54 @@ const struct Command Commands[] = {
   { "unscore",             mutt_parse_unscore,     0 },
   { "unset",               parse_set,              MUTT_SET_UNSET },
   { "unsetenv",            parse_setenv,           MUTT_SET_UNSET },
-#ifdef USE_SIDEBAR
-  { "unsidebar_whitelist", sb_parse_unwhitelist,   0 },
-#endif
   { "unsubjectrx",         parse_unsubjectrx_list, 0 },
   { "unsubscribe",         parse_unsubscribe,      0 },
-#ifdef USE_IMAP
-  { "unsubscribe-from",    parse_unsubscribe_from, 0 },
-#endif
-#ifdef USE_NOTMUCH
-  { "unvirtual-mailboxes", parse_unmailboxes,      0 },
-  { "virtual-mailboxes",   parse_mailboxes,        MUTT_NAMED },
-#endif
-  { NULL,                  NULL,                   0 },
 };
+
 // clang-format on
+
+ARRAY_HEAD(CommandsData, struct Command);
+struct CommandsData commands = ARRAY_HEAD_INITIALIZER;
+const struct Command *Commands;
```

Comment:
Why do you keep this around?

Reply:
I've eliminated the redundant stuff. Let me know if it looks better now.

Reply:
Uhm I'm still wondering why you need the global `Commands` pointer. Can't we have APIs to access the local `commands` array?

Reply:
As discussed with @gahr and @flatcap on IRC remove the global `Commands` array and provider further API to access the `commands` array.